packages/react-scripts/template/src/components/example/UseZionAxios.js

```diff
@@ -0,0 +1,16 @@
+import { i18n } from '@fs/zion-locale'
+import { authHeaderInterceptor } from '@fs/zion-axios'
+import { useAxios } from 'use-axios-client'
+
+export default function useZionAxios(url) {
+  let config = {
+    headers: {
+      common: {},
```

Comment:
does this intercepter need this? I find this setup a little strange. I'm wondering now if it's useful to expose this intermediate pattern to CRA. If it's this awkward, I'm perhaps inclined to not put this in CRA until we properly integrate it with zion-axios. This feels like cumbersome boilerplate to inflict on apps.

Reply:
Yeah this is the one part that feels icky.